Server/src/services/authenticationService.ts

```diff
@@ -0,0 +1,142 @@
+import * as jwt from 'jsonwebtoken';
+import * as argon2 from 'argon2';
+import 'dotenv/config';
+
+import { AuthenticationModel } from '../models/AuthenticationModel'
+
+import { ISignUpInformation } from '../genericInterfaces/AuthenticationInterfaces';
+import { ILoginInformation } from '../genericInterfaces/AuthenticationInterfaces';
+import { IJwtParams } from '../genericInterfaces/AuthenticationInterfaces';
+
+import { IResponse } from '../genericInterfaces/ResponsesInterface'
+
+
+/**
+ * This class services authentication or User related requests and handles
+ * interacts with the database for the Accounts table
+ */
+export class AuthenticationService {
+    private requestResponse: IResponse = {} as any;
```

Comment:
io dont think you need 'as any' in this case, why add it?

Reply:
I think it was breaking because there was no initialization

Reply:
Fixed wiith spread